tests/parametric/test_config_consistency.py

```diff
@@ -48,7 +48,7 @@ def test_tracing_disabled(self, library_env, test_agent, test_library):
 
 
 @scenarios.parametric
-@features.tracing_configuration_consistency
+@features.ddtrace_log_directory
```

Comment:
maybe trace_log_directory instead?

Reply:
I wasn't sure if the log directory was trace specific or if it was used by other ddtrace products. I can change the name